Makefile

```diff
@@ -3,6 +3,7 @@ all: libdragon
 V = 1  # force verbose (at least until we have converted all sub-Makefiles)
 SOURCE_DIR = src
 BUILD_DIR = build
+FILE_PREFIX=libdragon
```

Comment:
I think `FILE_PREFIX` is too generic. This is a very specific macro for a very specific thing, so I'd use a longer name. Something like `N64_BACKTRACE_FILE_PREFIX`.

I'd use the `N64` prefix because it's something related to N64 not your PC host, and we use the `N64` prefix on some/most of those macros.